sdk/eventgrid/azure-messaging-eventgrid-perf/src/main/java/com/azure/messaging/eventgrid/perf/ServiceTest.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.eventgrid.perf;
+
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.messaging.eventgrid.EventGridPublisherAsyncClient;
+import com.azure.messaging.eventgrid.EventGridPublisherClient;
+import com.azure.messaging.eventgrid.EventGridPublisherClientBuilder;
+import com.azure.perf.test.core.PerfStressOptions;
+import com.azure.perf.test.core.PerfStressTest;
+
+public abstract class ServiceTest<TOptions extends PerfStressOptions> extends PerfStressTest<TOptions> {
+    protected final EventGridPublisherClient cloudEventPublisherClient;
+    protected final EventGridPublisherAsyncClient cloudEventPublisherAsyncClient;
+
+    protected final EventGridPublisherClient eventGridEventPublisherClient;
+    protected final EventGridPublisherAsyncClient eventGridEventPublisherAsyncClient;
+
+    public ServiceTest(TOptions options) {
```

Comment:
Add some javadocs everywehre, just as best practice.

Reply:
Added javadocs (pretty simple though) to pass check style.